New abilities first commit (#36)
GameBrains/er-core: modules/experimental/pages/siege.adoc

= Siege

`Data sheet modifier`

These units are specially adapted from their original design, to specialise in close support of other units.
They have particular value in operations that involve assaults on fortifications or in the close confines of cities.

[NOTE.e40k]
====

> This data sheet modifier ability has been replaced with the more powerful {support} ability.

Reviewer: Change to active voice, like other similar cases — plus I'd argue that 'more flexible' is a better fit than 'more powerful' because the latter only makes sense in relation to base units that have a Firepower value of `1`:
```suggestion
We've replaced this ability with the more flexible {support} ability.
```
